catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -429,23 +433,39 @@ protected void showRenameDialog(List<T> selectedItems) {
 				.show();
 	}
 
-	protected void showMergeAlert() {
+	protected void showMergeDialog(List<T> selectedItems) {
 		if (adapter.getSelectedItems().size() <= 1) {
 			ToastUtil.showError(getContext(), R.string.am_merge_error);
 		} else {
-			ToastUtil.showError(getContext(), "These two projects cannot yet be merged");
+			TextInputDialog.Builder builder = new TextInputDialog.Builder(getContext());
+
+			builder.setHint(getString(R.string.project_name_label))
+					.setPositiveButton(getString(R.string.ok), (TextInputDialog.OnClickListener) (dialog, textInput)
+							-> {
+						mergeProjects(selectedItems, textInput);
+					});
+
+			builder.setTitle(R.string.new_project_dialog_title)
+					.setNegativeButton(R.string.cancel, null)
+					.show();
 		}
+
 		setShowProgressBar(true);
 		finishActionMode();
 	}
 
 	@StringRes
 	protected abstract int getRenameDialogTitle();
+
 	@StringRes
 	protected abstract int getRenameDialogHint();
 
 	protected void renameItem(T item, String newName) {
 		item.setName(newName);
 		finishActionMode();
 	}
+
+	protected void mergeProjects(List<T> selectedProjects, String mergeProjectName) {
+		finishActionMode();
```

Comment:
This should tell the user that the implementation is still missing, just closing the action mode looks like a bug

Reply:
Maybe just reuse the toast we had before...

Reply:
1. This is now misleading, as there is no actual merge happening
2. The string should be translated in `R.string.*` so it can be translated (even if it will be removed at a later point)